docs/cloud-resolver.md

```diff
@@ -95,50 +98,45 @@ GPU_RESOURCE_REQS = ResourceRequirements(
     )
 )
 
-@sematic.func(resource_requirements=GPU_RESOURCE_REQS)
+@sematic.func(standalone=True, resource_requirements=GPU_RESOURCE_REQS)
 def train_model(...):
     ...
 ```
 
 If there is a corresponding node available in your Kubernetes cluster, this
 function will be executed on that node.
 
-Note that `inline=False` is necessary for these resource requirements to be
+Note that `standalone=True` is necessary for these resource requirements to be
 honored, otherwise, they will be ignored.
 
-### Understanding "Inline"
+### Understanding Inline and Standalone Functions
 
-Before understanding inline functions, it is first helpful to refresh yourself
-on the description of the "driver" job above. Note that this job is distinct
-from the container where the root run executes - it's best to think of the
-driver as an "extra" container for the overall pipeline.
```

Comment:
Why remove this paragraph?

Reply:
Too verbose and unnecessary. We don't need to remind readers to read other parts of the docs.